fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckState.kt

```diff
@@ -158,7 +161,7 @@ private val fakeAnalysis = ReviewQualityCheckState.OptedIn.ProductReviewState.An
     needsAnalysis = false,
     adjustedRating = 3.6f,
     productUrl = "123",
-    highlights = mapOf(
+    highlights = sortedMapOf(
```

Comment:
how is it sorting the `enum` here? rather, by default, what does the sorting/ordering of enum values look like?

Reply:
It's in the order they are declared in the enum class